File: python/calculator.py
class Calculator: 
    def __init__(self, x, y): 
        self.x = x
        self.y = y

    def addition(self): 
        print(self.x + self.y)
    def substraction(self): 
        print(self.x - self.y) 
    def multiplication(self): 
        print(self.x*self.y)
    def division(self): 
        try: 
            print(self.x/self.y)
        except ZeroDivisionError: 
            print("Zero division not allowed")

c1 = Calculator(6, 8)
print(c1.addition())

File: php/Calculator.php
<?php 
Class Calculator{
    public $a; 
    public $b; 
    
    public function add($a, $b){
        return $a+$b;
    }
    public function substraction($a, $b){
        return $a - $b;
    }
    public function multiplication($a, $b){
        return $a*$b; 
    }
    public function division($a, $b){
        try{
            if ($a != 0 and $b != 0){
                return $a/$b;
            }
        } catch (Exception $ex) {
            echo $ex->getMessage();
        }
    }
    public function modulus($a, $b){
        return $a%$b;
    }
}
$c = new Calculator(); 
var_dump($c->add(2, 5));
var_dump($c->substraction(3, 1)); 
var_dump($c->multiplication(3, 5)); 
var_dump($c->division(1, 1)); 
var_dump($c->modulus(2, 2));